pkg/operator/status.go

```diff
@@ -243,6 +246,15 @@ func (optr *Operator) syncDegradedStatus(ierr syncError) (err error) {
 	return optr.updateStatus(co, coStatus)
 }
 
+type versionSkew int
+
+const (
+	skewUnchecked versionSkew = iota
+	skewSupported
+	skewUnsupported
+	skewPresent
+)
```

Comment:
Can you explain what's happening here with these const? Why not assign them the value of the Reason strings?